libfreerdp/core/gateway/rdg.c

```diff
@@ -161,20 +161,57 @@ static BOOL rdg_send_tunnel_request(rdpRdg* rdg)
 {
 	wStream* s;
 	BOOL status;
-	s = Stream_New(NULL, 16);
+	WCHAR* PAACookie = NULL;
+    UINT16 PAACookieLen = 0;
```

Comment:
Formatting, please check

Reply:
Thanks, modfied tabsize in my editor from 4 to 8. Hope this is the correct value.